test/js/samples/svelte-element-static/input.svelte

```diff
@@ -0,0 +1,3 @@
+<svelte:element this='a' class='outer'>
+	<svelte:element this='span' class='inner' />
+</svelte:element>
```

Comment:
I realized that https://github.com/sveltejs/svelte/pull/7938 doesn't test static svelte:element (e.g. `<svelte:element this="p">`) should be optimized.
So I added this test.